lightkurve/asteroseismology.py

```diff
@@ -0,0 +1,353 @@
+"""Defines the asteroseismology module"""
+from __future__ import division, print_function
+
+import copy
+import os
+import logging
+import warnings
+
+import numpy as np
+from matplotlib import pyplot as plt
+import astropy
+from astropy import units as u
+from astropy import constants as const
+
+# from . import PACKAGEDIR, MPLSTYLE
+log = logging.getLogger(__name__)
+
+__all__ = ['dnu_mass_prior', 'estimate_radius', '_get_radius_err', 'estimate_mass',
+            '_get_mass_err', 'estimate_logg','_get_logg_err']
```

Comment:
By convention, an underscore at the start of a function name means "this is a hidden helper function which a user is never supposed to interact with", so I would remove them from `__all__`, unless you think these functions should be accessible by users, in which case we'd remove the underscore prefix.

Reply:
Hi Geert,

Thanks for the comment, i'll fix this on the next commit!